frontend/app/components/contextual-alert.tsx

```diff
@@ -16,11 +16,11 @@ export function ContextualAlert(props: ContextualAlertProps) {
   const alertColor = type === 'warning' ? 'border-l-amber-700' : type === 'danger' ? 'border-l-red-700' : type === 'info' ? 'border-l-cyan-700' : 'border-l-green-700';
 
   return (
-    <div className={`relative pl-4 sm:pl-6`}>
+    <div className={`relative my-4 pl-4 sm:pl-6`}>
```

Comment:
Margin space should be set by the component's consumer based on the usage context.

```suggestion
    <div className="relative pl-4 sm:pl-6">
```

Reply:
`my-4` should be set outside of this component. The outside spacing depends on the usage context of the `ContextualAlert` component. It's the reponsiblity of the component consumer.

Reply:
> `my-4` should be set outside of this component. The outside spacing depends on the usage context of the `ContextualAlert` component. It's the reponsiblity of the component consumer.

@sebastien-comeau removed it also because only 1 alert will show